tests/store/api/conftest.py

```diff
@@ -180,6 +180,102 @@ def fixture_rml_store(store: Store, helpers: StoreHelpers) -> Store:
     return store
 
 
+@pytest.fixture(name="case_id_with_single_sample")
+def case_id_with_single_sample():
+    return "screamingtiger"
+
+
+@pytest.fixture(name="case_id_with_multiple_samples")
+def case_id_with_multiple_samples():
+    return "righteouspanda"
+
+
+@pytest.fixture(name="case_id_without_samples")
+def case_id_without_samples():
+    return "confusedtrout"
+
+
+@pytest.fixture(name="sample_id_in_single_case")
+def sample_id_in_single_case():
+    return "ASM1"
+
+
+@pytest.fixture(name="sample_id_in_multiple_cases")
+def sample_id_in_multiple_cases():
+    return "ASM2"
+
+
+@pytest.fixture(name="dummy_store")
+def fixture_dummy_store(
```

Comment:
Is there a better description of the store (like intended use or capability in the tests). Mock is preferred over dummy at least

Reply:
Yes, maybe ```store_with_multiple_cases_and_samples``` is better. I could not really find a suitable existing mock.

Reply:
👍